File: android/app/src/main/java/com/vegettable/app/model/SeasonalInfo.java
package com.vegettable.app.model;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class SeasonalInfo {
    @SerializedName("cropName")
    private String cropName;

    @SerializedName("category")
    private String category;

    @SerializedName("peakMonths")
    private List<Integer> peakMonths;

    @SerializedName("isInSeason")
    private boolean isInSeason;

    @SerializedName("seasonNote")
    private String seasonNote;

    public String getCropName() { return cropName; }
    public String getCategory() { return category; }
    public List<Integer> getPeakMonths() { return peakMonths; }
    public boolean isInSeason() { return isInSeason; }
    public String getSeasonNote() { return seasonNote; }
}

File: android/app/src/main/java/com/vegettable/app/model/SeasonalInfo.kt
package com.vegettable.app.model

import com.google.gson.annotations.SerializedName

class SeasonalInfo {
    @JvmField
    @SerializedName("cropName")
    val cropName: String? = null

    @SerializedName("category")
    val category: String? = null

    @JvmField
    @SerializedName("peakMonths")
    val peakMonths: MutableList<Int?>? = null

    @JvmField
    @SerializedName("isInSeason")
    val isInSeason: Boolean = false

    @JvmField
    @SerializedName("seasonNote")
    val seasonNote: String? = null
}
